src/compiler/AST/Common/Type.elm

```diff
@@ -156,11 +174,13 @@ letter int =
 
     maybeWrapParens (List Int) ( "List Int", state )
 
+
```

Comment:
for some reason these start piling up :sweat_smile: ideally there would be none between `maybeWrapParens` and `( "List Int)", state)`